lexos/helpers/definitions.py

```diff
@@ -5,3 +5,20 @@
 _WORD_BOUNDARY_REGEX_STR = r'\s'
 
 WORD_REGEX = re.compile(_WORD_REGEX_STR, re.UNICODE)
+
+
+def count_phrase_in_text(phrase: str, text: str):
```

Comment:
I think it would be nice if you can consider the fact that phrases starts or end with spaces.

Reply:
Its done in the last line of the function: count += len(text.split(" " + phrase + " ")) - 1

Reply:
I mean phrase like " test test ", this will never get matched.

Reply:
why would anyone put " test test " instead of "test test" in the dictionary?

Reply:
@Landaluce that is the point of robustness, you don't want your function only to work on a specific type of input. If `" test test"` is a valid phrase based on your definition, then it should be handled.

Reply:
You don't need to fix this right now, you can just leave them in the documentation, saying some specific cases will not be handled by this function